<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APUInformationDockWidget</class>
 <widget class="QDockWidget" name="APUInformationDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>312</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>312</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>APU Information</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>9</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>APU Cycle:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>APU Sequencer Mode:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="apuSequencerMode"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="apuCycle"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="square1">
       <attribute name="title">
        <string>Square1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Length Counter:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QProgressBar" name="lengthCounter1">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="format">
           <string>%v</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>DAC:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QProgressBar" name="dac1">
          <property name="maximum">
           <number>15</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string>%v</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="square2">
       <attribute name="title">
        <string>Square2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Length Counter:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QProgressBar" name="lengthCounter2">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string>%v</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>DAC:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QProgressBar" name="dac2">
          <property name="maximum">
           <number>15</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string>%v</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="triangle">
       <attribute name="title">
        <string>Triangle</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Linear Counter:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QProgressBar" name="linearCounter3">
          <property name="maximum">
           <number>127</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string>%v</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Length Counter:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QProgressBar" name="lengthCounter3">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string>%v</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>DAC:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QProgressBar" name="dac3">
          <property name="maximum">
           <number>15</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string>%v</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="noise">
       <attribute name="title">
        <string>Noise</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Length Counter:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QProgressBar" name="lengthCounter4">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string>%v</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>DAC:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QProgressBar" name="dac4">
          <property name="maximum">
           <number>15</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string>%v</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dmc">
       <attribute name="title">
        <string>DMC</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Length Counter:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="3">
         <widget class="QProgressBar" name="lengthCounter5">
          <property name="maximum">
           <number>4081</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textDirection">
           <enum>QProgressBar::TopToBottom</enum>
          </property>
          <property name="format">
           <string>%v</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>DAC:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="3">
         <widget class="QProgressBar" name="dac5">
          <property name="maximum">
           <number>127</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string>%v</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Sample Address:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="sampleAddr5"/>
        </item>
        <item row="4" column="3">
         <widget class="QCheckBox" name="irqEnabled5">
          <property name="text">
           <string>IRQ Enabled</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Sample Length:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="sampleLength5"/>
        </item>
        <item row="5" column="3">
         <widget class="QCheckBox" name="irqAsserted5">
          <property name="text">
           <string>IRQ Asserted</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Sample Position:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="samplePos5"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Sample Buffer:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="sampleBufferContents5"/>
        </item>
        <item row="3" column="3">
         <widget class="QCheckBox" name="sampleBufferFull5">
          <property name="text">
           <string>Full</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
